tests/test_dataset.py

```diff
@@ -7,7 +7,7 @@
 from fastapi.exceptions import ResponseValidationError
 
 from constants import JSONLD
-from tests.fixtures.datasets import dataset_test_data
+from tests.fixtures.datasets_oxigraph import dataset_test_data
```

Comment:
Perhaps when the transition to oxigraph is complete and we get rid of all the mentions of the stub, we can rename these files back to not having the "oxigraph" in the name since it would be redundant. But that would come later.